config/crd/bases/elbv2.k8s.aws_ingressclassparams.yaml

```diff
@@ -146,6 +148,43 @@ spec:
                 - internal
                 - internet-facing
                 type: string
+              securityGroups:
+                description: SecurityGroups defines the security groups to attach
+                  to all Ingresses that belong to IngressClass with this IngressClassParams.
+                properties:
+                  ids:
+                    description: IDs specify the resource IDs of security groups.
+                      Exactly one of this, `managedInbound`, or `tags` must be specified.
+                    items:
+                      description: SecurityGroupID specifies a security group ID.
+                      pattern: sg-[0-9a-f]+
```

Comment:
Can it also accept SG names just to keep consistent with the `alb.ingress.kubernetes.io/security-groups` annotation

Reply:
No, the field should be strongly typed. If you want to search by the `Name` tag, use the `tags` field using a tag key of `Name`